source/mail-access.rst

```diff
@@ -10,10 +10,12 @@ Webmail
 You can access your emails using the webmail interface at `webmail.uberspace.de 
 <https://webmail.uberspace.de>`_. Use your full email address and the corresponding password to login. This works with your ``@uber.space`` address as well as with any addresses using your :ref:`own domains <mail-domains>`.
 
-For your ``@uber.space`` address the password is the same as for SSH access, which you can set in your `dashboard
-<https://dashboard.uberspace.de/dashboard/authentication>`_.
+SystemMail address access
+-------------------------
 
-.. warning:: If you never set your password for SSH access it won't be possible to authenticate with the ``@uber.space`` address. You have to explicitly set the password for SSH access once after you have created your uberspace account.
+For your ``@uber.space`` address the password is the same as for SSH access, which you can set in your `dashboard <https://dashboard.uberspace.de/dashboard/authentication>`_.
+
+If you never set your password for SSH access it won't be possible to authenticate with the ``@uber.space`` address. You have to explicitly set the password for SSH access once after you have created your uberspace account.
```

Comment:
Maybe this is not a warning but a ``tip::``? I'm totally fine with adding the first sentence here to the box.